cmd/policy.go

```diff
@@ -4,33 +4,53 @@ import (
 	"context"
 	"fmt"
 	"os"
-	"strings"
 
+	"github.com/MakeNowJust/heredoc"
 	v1 "github.com/odpf/guardian/api/handler/v1"
 	pb "github.com/odpf/guardian/api/proto/odpf/guardian"
 	"github.com/odpf/guardian/app"
 	"github.com/odpf/guardian/domain"
+	"github.com/odpf/salt/printer"
+	"github.com/odpf/salt/term"
 	"github.com/spf13/cobra"
 )
 
-func policiesCommand(c *app.CLIConfig, adapter v1.ProtoAdapter) *cobra.Command {
+//  PolicyCmd is the root command for the policies subcommand.
+func PolicyCmd(c *app.CLIConfig, adapter v1.ProtoAdapter) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "policies",
-		Short: "manage policies",
+		Use:   "policy",
```

Comment:
any specific reason for changing the command to singular word?

Reply:
Makes it easier to use. Checked some other good clis for reference as well. e.g. github etc.

Reply:
@rahmatrhd What do you think?

Reply:
Yeah, as we discussed, we will use singular word and add the plural as the alias